package.json

```diff
@@ -171,8 +171,8 @@
   },
   "dependencies": {
     "@brave/brave-ui": "0.40.6",
-    "@brave/leo": "github:brave/leo#96fbe5c38ebea8e5d8cacc30a94ea32e92639cd6",
```

Comment:
Was this change to the Nala version intentional? 

This has reverted the last Nala bump - just noticed while trying to bump that the diff is much bigger than expected.

@cuba @petemill @AlanBreck